doc/WebSite/Abp-Session.md

```diff
@@ -95,19 +95,19 @@ as shown below:
         }
     }
 
-Use method returns an IDisposable and it **must be disposed**. Once the
+The Use method returns an IDisposable and it **must be disposed**. Once the
 return value is disposed, Session values are **automatically restored**
 the to previous values.
 
 #### Warning!
 
-Always use it in a using block as shown above. Otherwise, you may get
-unexpected session values. You can have nested Use blocks and they will
+Always use the AbpSession in a using block as shown above. Otherwise, you may 
```

Comment:
I mean the ``_session.Use(...)`` method should be used in a using statement. AbpSession itself has different properties/methods thos are not need to be used in a using block.